spec/factories/components.rb

```diff
@@ -1,13 +1,16 @@
 FactoryBot.define do
+
   factory :sp_component do
     component_type { COMPONENT_TYPE::SP }
     name { 'Test Service Provider' }
+#    team { create(:team) }
```

Comment:
you probably didn't want to leave these here ;)

Reply:
D'oh.  Now sorted (via force push).